src/common/snippets/tests/src/pass/softmax_decomposition.cpp

```diff
@@ -0,0 +1,122 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include "pass/softmax_decomposition.hpp"
+#include "common_test_utils/common_utils.hpp"
+#include "subgraph_softmax.hpp"
+#include "subgraph_lowered.hpp"
+
+#include "snippets/pass/softmax_decomposition.hpp"
+#include "snippets/pass/insert_load_store.hpp"
+#include "snippets/pass/insert_movebroadcast.hpp"
+#include "snippets/pass/insert_buffer.hpp"
+#include "snippets/pass/propagate_buffer_offset.hpp"
+#include "snippets/pass/convert_power_to_powerstatic.hpp"
+
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+std::string SoftmaxTests::getTestCaseName(testing::TestParamInfo<SoftmaxParams> obj) {
+    Shape inputShape;
+    int axis;
+    std::tie(inputShape, axis) = obj.param;
+    std::ostringstream result;
+    result << "IS=" << CommonTestUtils::vec2str(inputShape) << "_";
+    result << "Axis=" << axis << "_";
+    return result.str();
+}
+
+void SoftmaxTests::SetUp() {
+    const size_t count = 10;
+    manager.register_pass<ngraph::pass::InitNodeInfo>();
```

Comment:
We don't need to register `InitNodeInfo` here and on L62 because it is registered in base class

Reply:
You're right that it's registered in base class but we just override this method (it doesn't work like ctor). But I understand your point and I added `LoweringTests::SetUp()` call instead of explicit pass registration.